src/http.cc

```diff
@@ -2193,6 +2193,29 @@ HttpStateData::buildRequestPrefix(MemBuf * mb)
     return mb->size - offset;
 }
 
+void
+HttpStateData::refreshConnectionCloseFlag()
+{
+    if (request->flags.mustKeepalive)
+        flags.connectionClose = false;
+    else if (request->flags.pinned)
+        flags.connectionClose = !request->persistent();
+    else if (!Config.onoff.server_pconns)
+        flags.connectionClose = true;
+    else if (flags.tunneling)
```

Comment:
```suggestion
    else if (flags.tunneling) {
```

I know you are copying this code, but I think it should be reformatted to avoid dangerous style.